mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -1485,7 +1485,8 @@ pub fn spawn_electrum(
                 .ok_or(ERRL!("Couldn't retrieve host from addr {}", req.url))?;
 
             // check the dns name
-            try_s!(DnsNameRef::try_from_ascii_str(host));
+            // FIXME: Does this have to be a DNS name only?
+            try_s!(ServerName::try_from(host));
```

Comment:
so it has to be a dnsname only, is that a TLS related thing?

Reply:
It can be an ip is what I meant, host here https://github.com/KomodoPlatform/komodo-defi-framework/blob/251a270429b2796dde64be042e4251a042301e01/mm2src/coins/utxo/rpc_clients.rs#L1483-L1485 can be an ip according to the docs https://docs.rs/http/0.2.12/http/uri/struct.Uri.html#method.host
The `try_from` checks if it can parse the string to a dnsname, if not, it tries to parse it to an ip address.

Reply:
> It can be an ip is what I meant, host here

Aha so I just should remove the note there (and not revert to using dns-only parser). Maybe also change `dns_name` to something more neutral, `host` for example.